modules/http/server/src/test/scala/GreeterRestTests.scala

```diff
@@ -114,83 +117,135 @@ class GreeterRestTests extends RpcBaseTestSuite with BeforeAndAfter {
   "REST Service" should {
```

Comment:
REST Service Clients?

Reply:
Well each test involves a client and a server/service so I didn't want to pick sides. :-) Note that I used the word `Server` for the tests involving sending/receiving raw JSON and `Service` for the tests involving the client/server decoding layers; there may be a clearer way to mark the distinction.